src/legacy/components/TransactionConfirmationModal/index.tsx

```diff
@@ -81,7 +81,7 @@ export function TransactionConfirmationModal({
 
 function getWidth(hash: string | undefined, activityDerivedState: ActivityDerivedState | null): number {
   if (activityDerivedState && getActivityState(activityDerivedState) === 'filled') {
-    return 400
+    return 470
```

Comment:
In that case you can unify the return with line `89`

Change the if condition to:

`if((activityDerivedState && getActivityState(activityDerivedState) === 'filled') || !hash)`

then remove lines 86, 88 and 89, returning `850` in the default case

Reply:
Or, can we specify the 470 value specifically for this surplus modal? Not sure that's a specific 'activity'.